map.go

```diff
@@ -621,26 +634,15 @@ func (mi *MapIterator) Next(keyOut, valueOut interface{}) bool {
 		return mi.err == nil
 	}
 
-	mi.err = errIterationAborted
+	mi.err = xerrors.Errorf("%w", ErrIterationAborted)
```

Comment:
Good question, I'm following the last example from https://blog.golang.org/go1.13-errors#TOC_5.

Without the wrapping, users might do `err == ErrIterationAborted` instead of using `errors.Is`.